### PR TITLE
Make Login list parties

### PR DESCRIPTION
**Pull request by gaborh-da**

### CODE REVIEW

File: ui/src/config.ts [hunk 1]
```diff
@@ -9,12 +9,18 @@ export const ledgerId = isLocalDev ? "daml-ui-template" : host[0];
...
+    : 'https://' + apiUrl.join('.') + (window.location.port ? ':' + window.location.port : '') + '/data/' + ledgerId + '/';
 
 // Unfortunately, the development server of `create-react-app` does not proxy
 // websockets properly. Thus, we need to bypass it and talk to the JSON API
 // directly in development mode.
-export const wsBaseUrl = isLocalDev ? 'ws://localhost:7575/' : undefined;
+export const wsBaseUrl =
+    isLocalDev
+    ? 'ws://localhost:7575/'
+    : undefined;
```
Comment (richardkapolnai-da) [inline ui/src/config.ts:23]: just curiosity: why do you use this format style?

File: ui/src/pages/login/Login.tsx [hunk 2]
```diff
@@ -1,29 +1,80 @@
...
+    partiesPromiseMemo.then(ps => setParties(ps));
+    this.parties = parties;
+  }
+
+  getParties() {
+    return this.parties
+      .sort(this.compareByDisplayName)
+      .map(this.addSpacesBetweenWordsInDisplayName, this);
+  }
+}
```
Comment (richardkapolnai-da) [inline ui/src/pages/login/Login.tsx:61]: I think this should be copied to a separate file for easier re-use. Maybe `Util.tsx`, maybe sth else like `GetParties.tsx`.
what do you think?
or maybe
```
Util.tsx -> rename to Util/Misc.tsx
Util/GetParties.tsx
```
?

**Comment by richardkapolnai-da** [comment]
pls rebase to make it green

File: ui/src/pages/login/Login.tsx (hunk 2 above)
Comment (richardkapolnai-da) [reply ui/src/pages/login/Login.tsx:61]: nvm, let's leave it here

File: ui/src/config.ts (hunk 1 above)
Comment (gaborh-da) [reply ui/src/config.ts:23]: I cannot read it when it is in one line, it is very long.

Comment (gaborh-da) [reply ui/src/config.ts:23]: And I also have an irrational fear of lines longer than 100 characters.

Comment (richardkapolnai-da) [reply ui/src/config.ts:23]: it's only 74 chars long.
however, line 15 is 124 chars long. :)
I'm okay with both, you can leave it, was simply curious.

Comment (gaborh-da) [reply ui/src/config.ts:28]: yes, line 15 is still terrible

**Pull request merged**